### PR TITLE
chore: fix typos

### LINKED ISSUE #1009: [DOCS] remove grammatically error & spelling mistakes from code
### Issue Description

Hi, I want to work on the documentation related to grammatical error, and spelling mistakes, and also work on some changes that increase the build quality of the project. 
please assign me under gssoc23 label. 
Thank You

### Additional Context

_No response_

### Screenshots or Examples (if applicable)

_No response_

### Proposed Solution (optional)

_No response_

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by gulshanjakhon**
## #1009

Closes #1009

## Screenshots
After removing all grammatical errors:

## Note to reviewers

I have carefully review all the code and find these silly mistakes, its difficult for me to take Screenshot having one minor error, so I open a PR.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,10 +17,11 @@ Thank you for taking the time to contribute to our project. Please take a moment
 - [Node.js](https://nodejs.org/) is installed.
 
 ---
+
 ## How to Contribute 🤔
 
 - Look at the existing [**Issues**](https://github.com/rupali-codes/LinksHub/issues) or [**create a new issue**](https://github.com/rupali-codes/LinksHub/issues/new/choose)!
-- [**Fork the Repo**](https://github.com/rupali-codes/LinksHub/fork). Then, create a branch for any issue that you are working on. Finally, commit your work.
+- [**Fork the Repo**](https://github.com/rupali-codes/LinksHub/fork). Then, create a branch for any issue that you are working on. Finally, commit to your work.
```
Comment (CBID2) [inline CONTRIBUTING.md:24]: [Proofreading]
```suggestion
- [**Fork the Repo**](https://github.com/rupali-codes/LinksHub/fork). Then, create a branch for any issue that you are working on. Finally, start working! 😊
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @gulshanjakhon! :) I just made a proofreading suggestion. :)

**Comment by gulshanjakhon** [comment]
> 

hi @CBID2  please review now.

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Comment by gulshanjakhon** [comment]
please assign gssoc23 label

Thank You

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @gulshanjakhon! :) I just did another review of your pull request. There are some branch conflicts that need to be solved. Consider going to [Resolving a merge conflict on GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) learn how to do that.

**Comment by gulshanjakhon** [comment]
hey please check now

**Review by CBID2 — DISMISSED** [review]
LGTM! 😀

**Review by krishstwt — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -153,12 +153,12 @@ _View the list of all contributors [here](https://github.com/rupali-codes/LinksH
...
 ## License 📝
 
-LinksHub is licensed under the terms of MIT License. check out [LICENSE](https://github.com/rupali-codes/LinksHub/blob/main/LICENSE) for details.
+LinksHub is licensed under the terms of the MIT License. check out [LICENSE](https://github.com/rupali-codes/LinksHub/blob/main/LICENSE) for details.
 
 <a name="support"></a>
 
 ## Support ⭐
 
 _We would love to have you, feel free to open issues and pull requests and Don't forget to leave a star⭐_
```
Comment (rupali-codes) [inline README.md:162]:
```suggestion
```

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Comment by ujjawaltyagii** [comment]
> I'm just curious, why did you replace '*' with '-'? it does the same thing right?

@gulshanjakhon  ??
(also please resolve merge conflicts)

**Review by ujjawaltyagii — COMMENTED** [review]
@gulshanjakhon  ??
(also please resolve merge conflicts)

**Comment by gulshanjakhon** [comment]
Hi @ujjawaltyagii @rupali-codes @CBID2 @krishstwt please review 
Thank you

**Review by CBID2 — DISMISSED** [review]
LGTM!

File: database/other/podcasts.json [hunk 3]
```diff
@@ -1,4 +1,5 @@
 [
+
```
Comment (krishstwt) [inline database/other/podcasts.json:2]: this is a  empty line

```suggestion
```

File: database/other/github.json [hunk 4]
```diff
@@ -55,4 +55,4 @@
     "category": "other",
     "subcategory": "github"
   }
-]
+]
```
Comment (krishstwt) [inline database/other/github.json:58]:
```suggestion
]

```

File: database/frontend/online-code-editors.json [hunk 5]
```diff
@@ -104,4 +104,4 @@
     "category": "frontend",
     "subcategory": "online-code-editors"
   }
-]
+]
```
Comment (krishstwt) [inline database/frontend/online-code-editors.json:107]:
```suggestion
]

```

**Review by krishstwt — COMMENTED** [review]
formatting issues 👍🏻

**Review by CBID2 — APPROVED** [review]
LGTM!

**Comment by gulshanjakhon** [comment]
> formatting issues 👍🏻

resolve formatting issue

**Pull request merged**